pages/home.html

```diff
@@ -59,8 +59,11 @@ <h3 id="name"></h3>
     </div>
     <div class="row">
         <div class="col-md-1 margin"></div>
-        <div class="col-md-5 col-md-offset-5">
-            <div id="continue-batch">continue batch</div>
+        <div class="col-md-5">
+            <div id="new-batch">Add new batch</div>
+        </div>
+        <div class="col-md-5">
+            <div id="continue-batch">Continue batch</div>
```

Comment:
It doesn't need to be because this is going to be refactored anyway with my own code.